web/blocks.html

```diff
@@ -0,0 +1,157 @@
+<html>
+  <head>
+    <script language="javascript" src="js/blocks_rts.js"></script>
+    <script language="javascript" src="js/blocks_lib.js"></script>
+    <script language="javascript" src="js/blocks_out.js"></script>
+
+    <!-- JQuery -->
+    <script src="https://code.jquery.com/jquery-2.2.4.min.js"></script>
+
+
+    <link rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
+    integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7"
+    crossorigin="anonymous">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="css/bootstrap.min.css">
```

Comment:
Why do you need both the CDN and local version of this?  I'd prefer just the CDN version.